test/crud/unified/updateOne-let.json

```diff
@@ -136,7 +136,7 @@
       "description": "UpdateOne with let option unsupported (server-side error)",
       "runOnRequirements": [
         {
-          "minServerVersion": "3.6.0",
```

Comment:
Oh I found the answer in https://jira.mongodb.org/browse/DRIVERS-2210:
> The buggy test still may pass on some driver because the error thrown by server depends on order of field in update command